### DISCUSSION TITLE
"All hero selections are now blind" kills turbo mode

### DISCUSSION THREAD

**Discussion by Lovecraft34** [ValveSoftware/Dota2-Gameplay#7211] [General]
Does anyone know what was the reason for changing the selection of heroes in turbo mode? Valve mentions that it is a game mode that has gained popularity, however they make an absurd decision ruining a game mode, they may not understand if they don't play this mode, but imagine picking a hero and realizing that you are countered by 5 enemy heroes, sounds stupid right? Well, that's what now happens in this game mode, winning a game now is simply "luck", I've spent years spending time and money on this game and for the first time I'm considering leaving it. (I'm just a player who doesn't care about mmr, instead I prefer to just enjoy the hours of my life in a game and collect items).

**Comment by Maverickmax1** [comment]
Bro, what if someone picked something and you counter it with 5 heroes since you saw what they picked? I love this change.

**Reply by leamare** [reply]
I have to agree with you on that.
Though Turbo would benefit from a proper Ranked AP-like draft stage rather than blind pick.
